noirpacker.py

```diff
@@ -1,59 +0,0 @@
-#!/usr/bin/python3
-import os
-import re
-import sys
-
-def read_dir(path):
```

Comment:
I've created #951 to track discussions around noir-packer . Lets add back, then merge this PR and use that issue to decide if we should remove noirpacker.py